ci/unit_tests/utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import subprocess
+
+from monai.bundle import ConfigParser
+
+
+def run_command(cmd):
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+    while process.poll() is None:
+        line = process.stdout.readline()
+        line = line.rstrip()
+        if line:
+            print(line, flush=True)
+    print((f"Return code: {process.returncode}"))
+    process.stdout.close()
+
+
+def export_overrided_config(config_file, override_dict, output_path):
+    parser = ConfigParser()
+    parser.read_config(config_file)
+    for k, v in override_dict.items():
+        parser.set(v, k)
+    ConfigParser.export_config_file(parser.config, output_path, indent=4)
+
+
+def produce_mgpu_cmd(config_file, meta_file, logging_file, nnodes=1, nproc_per_node=2):
+    cmd = [
+        "torchrun",
+        "--standalone",
+        f"--nnodes={nnodes}",
+        f"--nproc_per_node={nproc_per_node}",
+        "-m",
+        "monai.bundle",
+        "run",
+        "--config_file",
+        config_file,
+        "--logging_file",
+        logging_file,
+        "--meta_file",
+        meta_file,
+    ]
+    return cmd
+
+
+def export_config_and_run_mgpu_cmd(config_file, meta_file, logging_file, override_dict, output_path, nnode=1, ngpu=2):
+    """
+    step 1: export a config file with overrided components
+    step 2: produce multi-gpu running command
+    step 3: run produced command
+    """
+    export_overrided_config(config_file=config_file, override_dict=override_dict, output_path=output_path)
```

Comment:
Please check properties after merging the train.json and multi_gpu_train.json.

Reply:
Think about it, maybe you can optimize the steps.

Thanks.

Reply:
to be discussed:
https://github.com/Project-MONAI/MONAI/blob/9d6ccce3d46d64b3ffe289349f50323bd0a1b6eb/monai/bundle/workflows.py#L381
related check may not be generic

Reply:
Update: I submitted a PR in: https://github.com/Project-MONAI/MONAI/pull/6560/files
Hi @Nic-Ma , I think in this PR, we can disable property checks on multi-gpu configs, what do you think?

Reply:
Can you run the checks for multi-gpu with your MONAI core PR now?

Thanks.

Reply:
Hi Nic, the latest MONAI core code works fine according to my local test. But in order to enable it, we may need a new tag. Otherwise, we have to do pip install on dev branch.

Reply:
We will release MONAI 1.2 soon, you can wait and use this PR to develop unit tests for more bundles.

Thanks.

Reply:
Sure, thanks Nic. I will prepare more tests on other bundles within this PR.

Reply:
Hi @Nic-Ma , considering we need to fix the multi-node issue, I think maybe we can merge this PR first (in order to enable unit tests), for the check properties on multi-gpu configs, it could be considered as an enhancement, and can be addressed in a following PR after MONAI 1.2 is released. What do you think?